pal/system-info/enum-type/feature.ts

```diff
@@ -107,9 +107,9 @@ export enum Feature {
      */
     EVENT_HANDHELD = 'EVENT_HANDHELD',
     /**
-     * @en Check whether Webassembly is supported at runtime. Generally, it needs to be checked when the constant `WASM_SUPPORT_MODE` is 2.
+     * @en Check whether Webassembly is supported at runtime. Generally, it needs to be checked when the constant `NATIVE_CODE_BUNDLE_MODE` is 2.
```

Comment:
`Check whether Webassembly is supported at runtime` the description is not correct now.

Reply:
It's correct. It's wasm support feature.
This comment says: This is a value for indicating whether the browser supports WASM. User needs to check this `WASM feature` when `NATIVE_CODE_BUNDLE_MODE` is `asmjs + wasm`.